test/extension/dynamic_image/any_image_view.cpp

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright 2020 Samuel Debionne
+//
+// Distributed under the Boost Software License, Version 1.0
+// See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt
+//
+#include <type_traits>
```

Comment:
Nitpick: I'd move this below `#include <boost/core/lightweight_test_trait.hpp>` as per suggested/consistent https://github.com/boostorg/gil/wiki/Include-Directives-Order

Reply:
Oupsy, sorry I usually include the other way around, from std then 3rd parties to mylib. I'll fix that.